courses/free-courses-te.md

```diff
@@ -28,6 +92,31 @@
 * [Full Java Script Tutorials in Telugu - Telugu Web Guru](https://www.youtube.com/playlist?list=PLh6Yk2rpZu2KqDjTuU_qHr-tI_CHOkIsN) - telugu web guru (YouTube)
 
 
+### JSON
+
+* [JSON ఒక గంటలో నేర్చుకోండి - JSON Crash Course - JSON Tutorial in Telugu](https://www.youtube.com/watch?v=DFPyRudVhOg) - Telugu Computer World (YouTube)
+
+
+### JQuery
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. use javascript